lib/bgsite_official_web/router.ex

```diff
@@ -132,6 +132,9 @@ defmodule BgsiteOfficialWeb.Router do
     put "/users/settings", UserSettingsController, :update
     get "/users/settings/confirm_email/:token", UserSettingsController, :confirm_email
     live "/tag_live/:id", WebsitesLike
+    get "/user/profile", UserController, :index
```

Comment:
Not related to this change, but, pls check what is the entry on row 101:
```
router paths should begin with a forward slash, got: "admin/websites"
```

Reply:
yeah, a TYPO.. fixed :+1: